graph/src/ipfs/retry_policy.rs

```diff
@@ -3,13 +3,10 @@ use std::time::Duration;
 use slog::Logger;
 
 use crate::ipfs::error::IpfsError;
+use crate::prelude::*;
 use crate::util::futures::retry;
 use crate::util::futures::RetryConfig;
 
-/// This is a safety mechanism to prevent infinite spamming of IPFS servers
-/// in the event of logical or unhandled deterministic errors.
-const DEFAULT_MAX_ATTEMPTS: usize = 10_0000;
-
```

Comment:
The default for the new env var raises the default from 10k to 100k. Is that intentional?

Reply:
i thought that `usize: 10_0000` is 100K as the underscore is ignored. should I use the same `10_000` inside the mappings?

Reply:
Oh wow .. I was fooled by the creative placement of the `_` - yes, `10_0000` is 100k. No need to change anything.